src/main/java/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.Bet;
+import model.Human;
+
+//создаём базу данных - хранилище
+
+public class Storage {
+
+    public static final  List<Bet> BETS = new ArrayList<>();
```

Comment:
```suggestion
    public static final List<Bet> BETS = new ArrayList<>();
```

Reply:
is it constant? Let's use camelCase)
the same with HUMANS

Reply:
done